### PR TITLE
Final adjustments to all celesta related files

### LINKED ISSUE #5041: new module: CELESTA
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module
```[tasklist]
### Tasks
```

### PR DESCRIPTION
**Pull request by LukasHats**
## PR checklist
Closes #5041  (https://github.com/nf-core/modules/issues/5041

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [x] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/celesta/main.nf [hunk 1]
```diff
@@ -0,0 +1,63 @@
+process CELESTA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "ghcr.io/schapirolabor/mcmicro-celesta:v0.0.2"
+
+    input:
+
```
Comment (kbestak) [inline modules/nf-core/celesta/main.nf:8]: remove empty line after input:

File: modules/nf-core/celesta/meta.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - cell_type_identification
+  - cell_phenotyping
+  - image_analysis
+  - mcmicro
+  - machine_learning
+tools:
+  - "celesta":
+      description: "Automate unsupervised machine learning cell type identification using both protein expressions and cell spatial neighborhood information"
+      homepage: "https://github.com/plevritis-lab/CELESTA"
+      documentation: "https://github.com/plevritis-lab/CELESTA/blob/main/README.md"
```
Comment (kbestak) [inline modules/nf-core/celesta/meta.yml:15]: add your repo with the Celesta CLI you've implemented for documentation instead here

File: modules/nf-core/celesta/meta.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test']`
+  - img_data:
+      type: file
+      description: "Quantification table with single cells as rows, marker intensities and X/Y coordinates as columns"
```
Comment (kbestak) [inline modules/nf-core/celesta/meta.yml:28]: Any specific column names here?

File: modules/nf-core/celesta/tests/main.nf.test [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "celesta"
+
+    test("celesta - csv") {
+
+        when {
+            process {
+                """
```
Comment (kbestak) [inline modules/nf-core/celesta/tests/main.nf.test:15]: remove TODOs

**Review by kbestak — COMMENTED** [review]
Minor adjustments mentioned

File: modules/nf-core/celesta/main.nf (hunk 1 above)
Comment (LukasHats) [reply modules/nf-core/celesta/main.nf:8]: done

**Comment by kbestak** [comment]
Since you don't have conda support, you have to remove the module from the automatic conda tests. You can do this by adding the module to the list of modules to be excluded in modules/.github/workflows/test.yml under the nf-test: section. Check spatiallda or backsub as examples.

**Review by kbestak — APPROVED** [review]
LGTM, all comments addressed!

**Pull request merged**